docs/sql-migration-guide.md

```diff
@@ -87,6 +87,8 @@ license: |
 
   - In Spark 3.1, the temporary view will have same behaviors with the permanent view, i.e. capture and store runtime SQL configs, SQL text, catalog and namespace. The capatured view properties will be applied during the parsing and analysis phases of the view resolution. To restore the behavior before Spark 3.1, you can set `spark.sql.legacy.storeAnalyzedPlanForView` to `true`.
 
+  - In Spark 3.1, temporary view created via `CACHE TABLE ... AS SELECT` will also have the same behavior with permanent view. In addition, when the temporary view is dropped, Spark will invalidate all cache dependents of the view, as opposed to just uncache the temporary view itself. To restore the behavior before Spark 3.1, you can set `spark.sql.legacy.storeAnalyzedPlanForView` to `true`.
```

Comment:
I hope "as opposed" doesn't mean that 3.1 invalidates only dependents but not the table itself.

Reply:
I intend to say it invalidates the cache for the temporary view itself as well as caches that depend on the view. Let me revise it.